skbio/io/tests/test_qseq.py

```diff
@@ -266,23 +265,29 @@ def test_invalid_files(self):
                     self.assertIn(e, str(cm.exception))
 
     def test_valid_files(self):
-        for constructor in [partial(Sequence), partial(DNA, validate=False),
-                            partial(RNA, validate=False),
-                            partial(Protein, validate=False)]:
+        for constructor in [Sequence, DNA, RNA, Protein]:
             for valid, kwargs, components in self.valid_files:
-                for kwarg in kwargs:
-                    _drop_kwargs(kwarg, 'constructor', 'filter')
-
-                    seq_num = kwarg.get('seq_num', 1)
+                for observed_kwargs in kwargs:
+                    expected_kwargs = {}
+                    # Currently not validating the alphabet for fastq
+                    # files that are read in.
```

Comment:
Mention that this is specific to the test, lest the reader extrapolate that statement to `fastq` generally.